lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -96,7 +96,13 @@ def word_symbol_pair?(pair)
           return false unless key.sym_type?
 
           sym_name = key.loc.expression.source
-          sym_name !~ /\A:["']|=\z/
+          valid_19_syntax_symbol?(sym_name)
+        end
+
+        def valid_19_syntax_symbol?(sym_name)
+          return false if sym_name =~ /\A:["']/
```

Comment:
Yup, there just seem to be a number of tests that more restrictive than they need to be for 2.2.